/**/admittance_controller_node:
  ros__parameters:
    mass:       5.0
    damping:   50.0
    stiffness: 100.0

/**/virtual_pose_publisher_node:
  ros__parameters:
    delay:     0.0
    frequency: 0.002
    x_pos:     0.0
    y_pos:     0.230
    z_pos:     0.067

/**/force_publisher_node:
  ros__parameters:
    delay:     0.0
    frequency: 0.002
    max_ticks: 2500
    duration:  120.0 # in seconds